src/R3.Unity/Assets/R3.Unity/Runtime/External/TextMeshPro/R3.Unity.TextMeshPro.asmdef

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "R3.Unity.TextMeshPro",
+    "rootNamespace": "",
+    "references": [
+        "R3.Unity",
+        "Unity.TextMeshPro"
+    ],
+    "includePlatforms": [],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
+    "defineConstraints": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.textmeshpro",
+            "expression": "",
+            "define": "R3_TEXTMESHPRO_SUPPORT"
+        },
+        {
+            "name": "com.unity.ugui",
+            "expression": "",
+            "define": "R3_UGUI_SUPPORT"
+        }
```

Comment:
If using com.unity.ugui 2.0.0 or later, you should still define R3_TEXTMESHPRO_SUPPORT as com.unity.textmeshpro has been integrated into com.unity.ugui starting from Unity 2023.2.

https://discussions.unity.com/t/2023-2-latest-development-on-textmesh-pro/917387

Reply:
Thank you for pointing it out. I have made the correction!